jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -1391,6 +1391,35 @@
       (pushback (send *baxter* arm :inverse-kinematics coords-tmp :rotation-axis t) avs)
       (send self :angle-vector-sequence avs :fast nil 0 :scale 4.0)
       ))
+  (:view-opposite-hand-pose
+    (arm)
+    (let (opposite-arm gripper-coords opposite-gripper-coords dy theta)
+      (if (eq arm :larm)
+        (progn
+          (setq opposite-gripper-coords
+                (send (send *baxter* :rarm-end-coords) :worldcoords)))
+        (progn
+          (setq opposite-gripper-coords
+                (send (send *baxter* :larm-end-coords) :worldcoords))))
+      (setq gripper-coords (send opposite-gripper-coords :copy-coords))
```

Comment:
L1397-L1404
`(setq gripper-coords (send (send *baxter* arm :end-coords) :copy-worldcoords))` is much shorter and easier to understand, I think